### ISSUE TITLE
Add Orthophotograph of Helsinki

### ISSUE THREAD

**Issue by theel0ja** [osmlab/editor-layer-index#339]
City of Helsinki's orthophotograph.

## URL of the imagery
https://www.avoindata.fi/data/en/dataset/helsingin-ortokuva-2014

WMS server: http://kartta.hel.fi/ws/geoserver/avoindata/wms
## Two letter country code
FI

## Desired layer name
Orthophotograph of Helsinki

## License
CC BY 4.0

## Attribution (text, URL, required?)
"Helsingin ortokuva 2014 by Helsingin kaupungin kiinteistövirasto is licensed under a Creative Commons Attribution 4.0 International License."

## Icon
(optional)

## Boundary polygon
(if possible, provide a boundary polygon, e.g. as a GeoJSON or OSM file.)

**Comment by theel0ja** [comment]
`waitfor - license permission` should be added, too.

**Comment by NKAmapper** [comment]
@theel0ja : The orthophotos provided by the Helsinki region seems to have a higher quality than the national MML orthophotos. They are open under the CC BY 4.0 license and distribution is actively encouraged: https://hri.fi/en_gb/helsinki-region-gis-data-freely-at-your-disposal/.

Would you mind asking for confirmation in an e-mail to hri.fi that their data is used in OSM?
Your request would need to mention that OSM is using the ODbL license and that attribution will be made at the https://wiki.openstreetmap.org/wiki/Contributors#Finland page.

I can then help adding the orthophotos for the region (Helsinki, Espoo, Vantaa, Kauniainen) to ELI and JOSM.

**Comment by NKAmapper** [comment]
#340

**Comment by grischard** [comment]
Thanks @theel0ja! As @NKAmapper rightly said, we need a confirmation that this data can be used in OSM. Unfortunately, CC-BY isn't fully compatible with OSM's ODbL, as that blogpost you linked to explains. Once you have a confirmation, I'll be very happy to add this.

**Comment by theel0ja** [comment]
@NKAmapper What should I ask in the email?

**Comment by NKAmapper** [comment]
This is the format we have used in Norway for most of our requests:

"I am one of many contributors to [OpenStreetMap](https:openstreetmap.org), a free, editable map of the whole world that is being built by volunteers and released with an open-content license.

We have seen that [Helsinki Region Infoshare](https:hi.fi) has released orthophotos and other open datasets under the CC BY 4.0 license. We would like to get your confirmation that these datasets may be used by OpenStreetMap.

OpenStreetMap is using the [ODbL](https://opendatacommons.org/licenses/odbl/) license for its data. Attribution to Helsinki Region Infoshare will be  made at [OSM Contributors](https://wiki.openstreetmap.org/wiki/Contributors#Finland)."

**Comment by theel0ja** [comment]
@NKAmapper Sent message to hri@hel.fi.

**Comment by theel0ja** [comment]
<img width="845" alt="screen shot 2018-08-07 at 16 03 11" src="https://user-images.githubusercontent.com/5832930/43777418-70e1a2f8-9a5b-11e8-9810-0169b85064aa.png">

**Comment by theel0ja** [comment]
Uploaded message as .eml to here:

https://storage.theel0ja.info/osm/VS%20Permission%20request%20for%20usage%20of%20datasets%20in%20OpenStreetMap.eml

**Comment by NKAmapper** [comment]
Great!
You now have the permission to use not only the orthophotos but also all the other open datasets from HRI.

I would recommend that you now post both your enquiry and the reply from HRI to the [Finnish OSM forum](https://forum.openstreetmap.org/viewforum.php?id=15) so that the permission is recorded there and that anyone may comment.

I will now prepare for getting the orthophotos into ELI and JOSM.

**Comment by theel0ja** [comment]
Done. https://forum.openstreetmap.org/viewtopic.php?pid=710304

**Comment by NKAmapper** [comment]
#532

**Comment by grischard** [comment]
Closed in #532, thanks to all those involved!

**Comment by houtari** [comment]
I believe this didn't go quite according to the copyright regulations. Firstly HRI isn't the owner of any of these dataset and thus isn't eligible to give out any rights associated with the datasets. The municipial orthophotos are owned by the municipalities (Helsinki, Espoo etc.) themselves and the Helsinki Region orthophoto is joint acquisition by HSY, HSL, Helsinki, Espoo and other municipalities so there are multiple owners of that datasets. See https://forum.openstreetmap.org/viewtopic.php?pid=710304 for more information.

**Comment by grischard** [comment]
Hi Markku!

Thanks for bringing this to our attention. I see three things here:

- What should the right name and attribution be?
- HRI looks like the correct point of contact for this kind of inquiries to me. @theel0ja was explicit about OpenStreetMap being licensed under the ODbL. Ms Lahti at HRI is the project manager of HRI and works at the City of Helsinki Executive Office - in other words, she's not just any random clown. Have you been in touch with her to clear this up?
- Could you or Henna-Kaisa from HSY please see about quickly getting us an unambiguous permission like the one you linked in the forum?

**Comment by theel0ja** [comment]
<img width="640" alt="screen shot 2018-09-07 at 20 56 27" src="https://user-images.githubusercontent.com/5832930/45235115-85ae2100-b2e0-11e8-8922-43a4a0202baa.png">

posting this email I got from @NKAmapper today (11:13 [EEST](https://en.wikipedia.org/wiki/Eastern_European_Summer_Time))

**Comment by houtari** [comment]
Hello all. Answers to your questions below...

**What should the right name and attribution be?**
- The required attribution depends on which photo set was used? Could you or someone else please specify which orthophoto was used in this case? If it was ([this one](https://hri.fi/data/fi/dataset/paakaupunkiseudun-ortokuva-2017)) then the attribution question was already answered by Henna-Kaisa@HSY [here](https://forum.openstreetmap.org/viewtopic.php?pid=714413#p714413). The name for the photo could then be eg. "Capital Area Ortophoto 2017"

**HRI looks like the correct point of contact for this kind of inquiries to me. @theel0ja was explicit about OpenStreetMap being licensed under the ODbL. Ms Lahti at HRI is the project manager of HRI and works at the City of Helsinki Executive Office - in other words, she's not just any random clown. Have you been in touch with her to clear this up?**
- I had a meeting at HRI (with Ms Lahti and others) on wednesday 5.9. where I came aware of this "HRI ortophoto" in ID editor. This raised some questions at the meeting so I started to investigate what it was all about. After that I've mailed Lahti and others but haven't had the opportunity to discuss this issue with them.
- I know Ms Lahti and indeed know she's no "random clown" :-) However everything isn't always what it looks like. HRI does indeed help to open up a lot of data. But they're seldom (or never?) the owner of the data themselves. Their main focus is to promote open data eg. by creating this open data catalog for the region where data from many different copyright owners (municipalities, cities and other govermental authorities) are collected and distributed. @theel0ja was indeed explicit about OpenStreetMap being ODbL based but to me it looked like HRI's answer was more or less a standard answer containing basic CCBY40-licence instructions like "the original data source should be mentioned somewhere". My guess is that they didn't quite understand what the "OSM-part" of the question requires. I'll ensure this thing on monday by contacting HRI. Furthermore it looked to me like @theel0ja misinterpreted HRI's message (example) regarding the right copyright attribution just as I already mentioned in my OSM-forum message.

**Could you or Henna-Kaisa from HSY please see about quickly getting us an unambiguous permission like the one you linked in the forum?**
- HSY was the main administrator of this ortophoto project so I would pass the task to them. But It probably requires acceptance from all parties involved - so I can't promise any quick response. This list of involved parties includes municipalities that aren't part of the HRI steering group so I can't see how the steering group could be authorized by the data owners as suggested by @NKAmapper.

**Comment by houtari** [comment]
After finding [https://josm.openstreetmap.de/wiki/Maps/Finland](https://josm.openstreetmap.de/wiki/Maps/Finland) and the URL-line with the link [https://kartta.hsy.fi/geoserver/ows?SERVICE=WMS&FORMAT=image/jpeg&TRANSPARENT=TRUE&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=taustakartat_ja_aluejaot:Ortoilmakuva_2017&STYLES=&SRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}](https://kartta.hsy.fi/geoserver/ows?SERVICE=WMS&FORMAT=image/jpeg&TRANSPARENT=TRUE&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=taustakartat_ja_aluejaot:Ortoilmakuva_2017&STYLES=&SRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}).  I would say my guess about the photo being the "Capital Area Ortophoto 2017" was spot on.  So the attribution would be ... _"Pääkaupunkiseudun ortoilmakuva (C) Espoon, Helsingin ja Vantaan kaupungit, Kirkkonummen ja Nurmijärven kunnat sekä HSL ja HSY, 2017"_ and the copyright owners : Espoo, Helsinki, Vantaa, Kirkkonummi, Nurmijärvi, HSL & HSY.

**Comment by houtari** [comment]
I just phoned Tanja Lahti @ HRI - she confirmed that my assumption (below) was right.

> HRI's answer was more or less a standard answer containing basic CCBY40-licence instructions like "the original data source should be mentioned somewhere". My guess is that they didn't quite understand what the "OSM-part" of the question requires. I'll ensure this thing on monday by contacting HRI.

**Comment by houtari** [comment]
This case seems still to be open and at least ID-editor still claims these to be HRI ortophotos. Could someone please change the name and attribution of the photo to the ones i presented above (7 sep)?

**Comment by theel0ja** [comment]
@houtari Latest stable release of iD might have old version of this list.

**Comment by theel0ja** [comment]
In development version, it has been changed.

http://preview.ideditor.com/master/#background=hri-orto&disable_features=boundaries&map=14.62/60.1908/24.9075

**Comment by NKAmapper** [comment]
Correct, the name and attribution text were updated in ELI a month ago: [#554 diff](https://github.com/osmlab/editor-layer-index/commit/17bf556c332b4724bb6d36c0bbff0b5a93db436b#diff-617a330fcea44751e97afaab10982699)

However, iD has not had a new release since 29 August. I could ask over at [iD](https://github.com/openstreetmap/iD) if they could do a special update for this imagery.

**Comment by NKAmapper** [comment]
"End of October" seems to be the timeline: [iD #5412](https://github.com/openstreetmap/iD/issues/5412#event-1912341090)

**Comment by grischard** [comment]
If only we could ask @bhousel when the next version of iD will be released :)

**Comment by houtari** [comment]
Now the attributions seems to be updated to the production releases of ID & JOSM. So, this task could probably be closed unless we wouldn't like to add another ortophoto? :-) This 5 cm resolution ortophoto over Helsinki city is far more detailed than this regional ortophoto (20 cm). Could this be added to the list as well? I believe it would be largely appreciated in the OSM community. I can ask for the needed permissions from the city as I allready have contacts there. But first could someone confirm that ID & JOSM can use this WMS-service? info : https://www.avoindata.fi/data/fi/dataset/helsingin-ortokuva-2018  ... WMS : https://kartta.hel.fi/ws/geoserver/avoindata/wms

**Comment by NKAmapper** [comment]
Yes, the WMS works, and I agree that the quality is much better.
Here is the url: https://kartta.hel.fi/ws/geoserver/avoindata/ows?SERVICE=WMS&FORMAT=image/jpeg&TRANSPARENT=TRUE&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=avoindata:Ortoilmakuva_2018_5cm&STYLES=&SRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}

I can help in adding it to JOSM and ELI (iD) if you get the permission and post it to the Finland forum. You may use the same wording in your request as shown earlier in this thread. You may want to phrase your request so that OSM gets the permission for all their orthophotos (not just 2018), so that this process does not need to be repeated next year. You may even want to ask them for permission to use all their open data in one go (this is what I usually do).

I believe this quality is available also from some of the other larger cities in Finland, if you would be interested in asking for even more permissions (or you could ask others at the forum).

**Comment by houtari** [comment]
Hello again! Sorry for the delay in answering. I believe that Helsinki is quite unique in Finland to offer this accurate aerial imagery as open data. I haven't heard of this kind of service from other cities. I'll push this issue foward this week by contacting the city of Helsinki and try to get their permission to this - which I believe will be quite easy as I already had some discussions about this last year and then they already agreed to give the permission or actually wodered why I even asked them for it. For them it was quite clear that if the OSM-editor just presented the copyright-attribution according to CCBY40-regulations then the new objects created with the editor could be licenced as OdBL according to OSM. I'll use a modified version of the request used earlier and ask the permissions for all their ortophotos. But I won't ask the permission for all their data. That I believe will complicate things to much as they publish open data that they're not the only copyright-owner too (as we earlier experienced with the HRI ortophoto) and they also publish vector data. With vector data we've used a different "permission scheme" at least here in Finland. With Vector data organizations have issued a permission like this one ... https://wiki.openstreetmap.org/w/images/7/75/HSL_permission.pdf

**Comment by grischard** [comment]
It's great that you're still on this, thank you :)

**Comment by houtari** [comment]
-----Alkuperäinen viesti-----
Lähettäjä: houtari 
Lähetetty: 13. helmikuuta 2019 13:15
Vastaanottaja: Tolkki Timo
Aihe: Permission to use Helsinki City ortophotos with OSM

Hello!

I am one of many contributors to OpenStreetMap, a free, editable map of the whole world that is being built by volunteers and released with an open-content license.


We have seen that Helsinki City has released an ortophoto covering whole of Helsinki with 5 cm resolution under the CC BY 4.0 license.

https://www.avoindata.fi/data/fi/dataset/helsingin-ortokuva-2018

We would like to get your confirmation that this and also other later on released CC BY 4.0-licensed ortophotos from Helsiki City may be used in the different OpenStreetMap-editors.

OpenStreetMap is using the ODbL
(https://opendatacommons.org/licenses/odbl/) license for its data. 
Attribution to Helsinki City ortophotos in accordance with the CC BY 4.0 license will naturally be presented when publishing the ortophotos in the OpenStreetMap-editors.

Sincerely,
Markku Huotari

**Comment by houtari** [comment]
-----Alkuperäinen viesti----- 
From: Tolkki Timo 
Sent: Thursday, February 14, 2019 4:21 PM 
To: houtari
Subject: VS: Permission to use Helsinki City ortophotos with OSM 

Markku,

https://creativecommons.org/licenses/by/4.0/deed.en CC BY 4.0 summary says:

"You are free to:
Share - copy and redistribute the material in any medium or format 
Adapt - remix, transform, and build upon the material for any purpose, even commercially. "

"Under the following terms:
Attribution - You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.  
No additional restrictions - You may not apply legal terms or technological measures that legally restrict others from doing anything the license permits."

Best regards,

Timo Tolkki 
Head of Unit

**Comment by houtari** [comment]
I guess this qualifies as a permission? :-)

**Comment by NKAmapper** [comment]
It seems that Timo has not actually granted any permission to OSM but merely quoted some of the key terms of CC BY 4.0. As explained [here](https://blog.openstreetmap.org/2017/03/17/use-of-cc-by-data/), Timo's last quote about additional restrictions is actually a problem because ODbL is more strict than CC BY 4.0 regarding the use of DRM.

What is needed from Timo is either a confirmation to your request, which would then be an explicit permission for OSM to use the orthophotos under ODbL. Such a confirmation may also be given in form of a waiver similar to the PDF which you have linked to a couple of messages earlier here.

**Comment by houtari** [comment]
OK! Thanks for the comments. I'll try again.

**Comment by simonpoole** [comment]
> As explained [here](https://blog.openstreetmap.org/2017/03/17/use-of-cc-by-data/), Timo's last quote about additional restrictions is actually a problem because ODbL is more strict than CC BY 4.0 regarding the use of DRM.

Small nitpick, it is the other way around: CC BY 4.0 is more restrictive than the OBbL wrt DRM (and a couple of other points).

**Comment by houtari** [comment]
I'm sorry if I'm repeating things that's already been explained somewhere here (or in the refered documents) but in order to further clarify things and to be able to explain in detail to the city of Helsinki why we're actually asking them for permission I would like to ask a couple of questions.

Just as background - the city of Helsinki at the moment interprets the license issued with their aerial photos (CCBY40) in such a manner that adding the copyright attribution by the photos in the OSM-editors fulfills any obligations the license requires. The objects created in the OSM-editors on top of / with the help of these aerial photos can thereafter according to Helsinki be licensed under ODbL without even needing to ask the city of Helsinki for a special permission. That's also why they thought quoting CC BY 4.0 was an adequate answer to my enquries. I guess this is the thing that's being interpreted differently in the OSM Licensing Working Group? I believe the OSM LWG see this data derived from the aerial photos as automatically CC40-licensed unless stated otherwise, or am I right? Do you think Helsinki has misinterpreted the license in this case or can there be different enforcement of the license in different countries? I guess not or else you would call it something else than CC BY 4.0 INTERNATIONAL?

With vector data it's somehow easier to understand and explain that data imported to OSM are fundamentally licensed under the original license, but is there actually legally any difference between imported vector data or this kind of derived data from aerial photographs and if not could you pinpoint to the legal text or license paragraphs where this is explained.

**Comment by simonpoole** [comment]
> 
> I guess this is the thing that's being interpreted differently in the OSM Licensing Working Group? I believe the OSM LWG see this data derived from the aerial photos as automatically CC40-licensed unless stated otherwise, or am I right? 

CC BY does not cater explicitly for this case so it boils down to territorial legislation etc. if the use of imagery to derive geometries is use covered by copyright or not. 

Definitely it is not something that can be ruled out based on first principles (the use of a creative work licence to protect the imagery in question would indicate that the rights owners are of the opinion that it is a work covered by copyright), so we need clarification. The alternative would be for us to get per territory formal legal advice, which is not going to happen for obvious reasons.

**Comment by houtari** [comment]
OK! Thanks for this clarification.

**Comment by houtari** [comment]
One more aspect to this question. If the ortophoto would be licensed CC0  - could we then use the ortophoto as it is without needing to separately ask for a permission? This could be a relevant option to the municipality as they would like to avoid this kind of "unnecessary bureaucracy" that possibly could be avoided by licensing the product differently.

**Comment by bhousel** [comment]
> If the ortophoto would be licensed CC0 - could we then use the ortophoto as it is without needing to separately ask for a permission? This could be a relevant option to the municipality as they would like to avoid this kind of "unnecessary bureaucracy" that possibly could be avoided by licensing the product differently.

Yes!  If they are willing to release their imagery into the public domain (CC0) that makes it the most broadly usable.

**Comment by NKAmapper** [comment]
This (CC0) is the route chosen in Sweden now, after realising that they did not follow up attribution anyway.
https://www.lantmateriet.se/sv/nyheter-och-press/Pressmeddelande/2017/lantmateriet-forenklar-villkoren-for-oppna-geodata/

**Comment by grischard** [comment]
We picked CC0 for similar reasons in Luxembourg: https://data.public.lu/fr/datasets/bd-l-ortho-webservices-wms-et-wmts/

**Comment by houtari** [comment]
> Yes! If they are willing to release their imagery into the public domain (CC0) that makes it the most broadly usable.

Thanks. Just as I thought. Wanted to check with you just to be sure. Thanks also for the references from Sweden and Luxembourg.

**Comment by grischard** [comment]
@houtari Happy to have a chat about how we ended up going CC-0 in Luxembourg - email is in my profile.

**Comment by xkr47** [comment]
Any progress on this one? Can it be used? If not, maybe it should not be in JOSM etc?

**Comment by houtari** [comment]
Unfortunately no progress with the Helsinki 5cm ortophoto from 2018 https://www.avoindata.fi/data/en_GB/dataset/helsingin-ortokuva-2018 , but that's not the one that's in JOSM etc. This one is https://hri.fi/data/en_GB/dataset/paakaupunkiseudun-ortokuva-2017 (Helsinki Region 20cm ortophoto from 2017) and for this one we have permission. The status with the first one is still the same. The city is sticking with CCBY40 but won't issue any additional permit required by OSM to make it available in JOSM etc, So we're kind of stuck on that one. 

Late last year a new Helsinki Region 20cm ortophoto was published. At the moment I'm gathering strength and time to go through the permit procedures with that one - these processes can be a bit tiring. :-) Hopefully I'll get back to this in the near future.

**Comment by andrewharvey** [comment]
In case you didn't know, the CC BY 4.0 waiver is at https://drive.google.com/file/d/0B3PN5zfbzThqeTdWR1l3SzJVcTg/view, is that the one they rejected?

**Comment by houtari** [comment]
I do know. Our organization has already been using it (https://wiki.openstreetmap.org/w/images/7/75/HSL_permission.pdf) for our own purposes. It's not so much that they (City of Helsinki) reject or disagree with any existing waiver, more that they just simply consider CCBY40-statements to be all they need to offer, refusing to give any additional permissions. "It's complicated" as Facebook would say.

**Comment by finfly** [comment]
I did research on legal side of the issue, while not being a layer literature research is familiar to me. Writing on this issue in English is beyond my abilities, thus I wrote in Finnish to here: https://forum.openstreetmap.org/viewtopic.php?pid=782920#p782920
